server/models/meshery_filter.go

```diff
@@ -21,7 +21,7 @@ type MesheryFilter struct {
 	Location    sql.Map `json:"location"`
 	Visibility  string  `json:"visibility"`
 	CatalogData sql.Map `json:"catalogData"`
-	Config      string  `json:"config"` 
+	FilterResource      string  `json:"filter_resource"` 
```

Comment:
`Component`?

Reply:
To be in accordance with how pattern_resources are stored, I chose `filter_resource`, as it is actually a instance of that component/ resource of that

Reply:
can we call it `component_resource`?

So that we can normalize this construct between patterns, filters etc.